app/models/client.rb

```diff
@@ -72,4 +72,8 @@ def self.first_languages
   def self.state_collection
     STATES.map(&:to_sym)
   end
+
+  def german_missing?
+    language_skills.blank? || language_skills.german.blank?
```

Comment:
Why is the first check needed?

Reply:
Your right, its a relict from a version where there was also a default empty native_speker language added. But now this is obsolete. Going to remove it.